README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A Little bit about me.
+
+Hello my name is Arturo Juarez. I am studying **International Relations** under the Committee of International Relations at the *University of Chicago*. I was born in California but I grew up in both Illinios and and Southern California. As such I consider myself both a Midwesterner and a Californian. My interests incluce reading, outdoor running, and traveling to new destinations both domestically and internationally. 
+
+### More about me
```

Comment:
Good use of different heading levels to show when you were getting more specific!

Reply:
^Agreed. I also like how you used different heading levels to introduce each paragraph. Maybe adding the main title at the very beginning that says "#A Short Self-introduction?" I know it's not necessary, but I'm wondering why you're using ### for each sub-title. (like, why three #? wouldn't 2 be enough?)

Reply:
^My apologies for the comment above! I didn't realize that you actually had a subtitle for this introductory paragraph that starts with ## A Little Bit About Me. Just weirdly the 3-5 lines are displayed after the 5-7 lines on my screen, so I didn't catch it at first sight. I'd say I really like the way you're experimenting with different levels of titles, and I think everything has come through perfectly :)